frontend/app/routes/_gcweb-app.personal-information.home-address.suggested.tsx

```diff
@@ -0,0 +1,154 @@
+import type { ReactNode } from 'react';
+
+import { type ActionFunctionArgs, type LoaderFunctionArgs, json, redirect } from '@remix-run/node';
+import { Form, Link, useLoaderData } from '@remix-run/react';
+
+import clsx from 'clsx';
+import { useTranslation } from 'react-i18next';
+
+import { Address } from '~/components/address';
+import { InputRadios } from '~/components/input-radios';
+import { getAddressService } from '~/services/address-service.server';
+import { sessionService } from '~/services/session-service.server';
+import { userService } from '~/services/user-service.server';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('personal-information');
+
+export const handle = {
+  breadcrumbs: [
+    { labelI18nKey: 'personal-information:home-address.suggested.breadcrumbs.home', to: '/' },
+    { labelI18nKey: 'personal-information:home-address.suggested.breadcrumbs.personal-information', to: '/personal-information' },
+    { labelI18nKey: 'personal-information:home-address.suggested.breadcrumbs.suggested-address' },
+  ],
+  i18nNamespaces,
+  pageIdentifier: 'CDCP-0008',
+  pageTitleI18nKey: 'personal-information:address.suggested.page-title',
+};
+
+/**
+ *
+ * loader, runs before rendering html code
+ *
+ */
```

Comment:
**Nitpick:** I feel we don't need this comment

Reply:
done